torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -36,6 +36,31 @@ std::string FusionExecutor::getStructuredCode(const std::string& kernel) {
   return code;
 }
 
+void FusionExecutor::compileFusionFromStr(
```

Comment:
Are these only used for debugging? I don't see any real code using them. Pointing it out in case this is accidentally checked in.